api/v1alpha1/application_types.go

```diff
@@ -219,6 +219,16 @@ func GetScalingReplicas(jsonReplicas *apiextensionsv1.JSON) (Replicas, error) {
 	return result, err
 }
 
+func IsHPAEnabled(jsonReplicas *apiextensionsv1.JSON) bool {
+	replicas, err := GetScalingReplicas(jsonReplicas)
```

Comment:
Just checking that you've taken into account the new replicas syntax that Johannes added in #280 where you're accessing it.

Reply:
This is the same function that Johannes used, just moved from `util` package to the API to avoid circular dependencies in packages. SKIPJobs do not have replicas, so I believe all should be good :)